packages/plugins/Traderxyz/src/SNSAdaptor/PostPreview.tsx

```diff
@@ -0,0 +1,332 @@
+/* eslint-disable no-restricted-imports */
+/* eslint-disable spaced-comment */
+/* eslint-disable eqeqeq */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
plz do not disable eslint rule

Reply:
Hi @septs, i have removed all the disable eslnt in following [commit](https://github.com/DimensionDev/Maskbook/pull/6120/commits/a1cd883b6856b69043fd54b510f22605caf7bf47) as seen below for example 

<img width="867" alt="image" src="https://user-images.githubusercontent.com/3579069/165238503-7ab14116-b23e-40dd-8061-cb856bda7675.png">